x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -80,6 +86,15 @@ export function hitsToGeoJson(hits, flattenHit, geoFieldName, geoFieldType) {
       geoShapeToGeometry(properties[geoFieldName], tmpGeometriesAccumulator);
     }
 
+    // There is a bug in Elasticsearch API where epoch_millis are returned as a string instead of a number
```

Comment:
I'd try and avoid heap-allocation in tight loops. This creates a new closure for each individual feature. Replace with simple for-loop.